docs/docs/miscellaneous/upgrading-from-yap-1x-to-yap-2x.md

```diff
@@ -10,6 +10,6 @@ parent: Miscellaneous
 ---
 
 
-Follow the upgrade steps as you normally do above.  You will also need to follow item #3 & #4 under [Setup](#setup), to add the Twilio credentials.
+Follow the upgrade steps as you normally do above.  You will also need to follow item #3 & #4 under [https://yap.bmlt.app/docs/general/setup/](https://yap.bmlt.app/docs/general/setup/), to add the Twilio credentials.
```

Comment:
Can these be relative links?

Reply:
damn it, i knew you were gonna ask that. Should of just done that in first place. I'm pretty sure they can be. ill check